src/hl.h

```diff
@@ -224,8 +224,8 @@ typedef wchar_t	uchar;
 #	define ustrlen		wcslen
 #	define ustrdup		_wcsdup
 HL_API int uvszprintf( uchar *out, int out_size, const uchar *fmt, va_list arglist );
-#	define _utod(s,end)	wcstod(s,end)
-#	define _utoi(s,end)	wcstol(s,end,10)
+#	define utod(s,len,end)		wcstod(s,end)
+#	define utoi(s,len,end)		wcstol(s,end,10)

```

Comment:
Why are we changing defines here?

Reply:
`_utod` and `_utoi` were used in actual `utod` and `utoi` functions which wrapped them and did an extra check ([see changes at the top of ucs2.c](https://github.com/HaxeFoundation/hashlink/pull/532/files#diff-b0ff64e049f3f8451842a16be609465714182f99675ae0c04b9d2940b0f6740cL29-L41)), however this check is no longer needed. Now they are direct replacements to get the function names the same between Windows and other systems, which handle strings differently. They were only used in one place, and this is how it was before 8d7b29c1865.

And the `len` was added because an extra parameter is needed in the non Windows version.

Reply:
Actually, sorry, the `len` should be removed from both versions as in the end it's not actually used.